javascript/javascript1/week2/weather_Wear.js

```diff
@@ -0,0 +1,27 @@
+
+function Temperature (tempToDay)
+{
+    if((tempToDay < -30) || (tempToDay > 51))
```

Comment:
if the temperture is higher then 51 then it's not really cold. Maybe you want to do something like
```suggestion
    if(tempToDay < -30)
```